bootstrapform_jinja/templatetags/bootstrap.py

```diff
@@ -1,110 +1,148 @@
-from django import forms
+from django.forms import CheckboxInput, CheckboxSelectMultiple, \
```

Comment:
I don't think we document it here, but we always use 120 max line length and parenthesis when imports span multiple lines/